QuizGame/AppShell.xaml

```diff
@@ -9,6 +9,6 @@
     <ShellContent
         Title="Home"
         ContentTemplate="{DataTemplate local:MainPage}"
-        Route="MainPage" />
+        Route="Home" />
```

Comment:
This is to stop an error when navigating back to `nameof(MainPage)` from results view